web/elm/tests/Common.elm

```diff
@@ -65,6 +67,18 @@ notContains x xs =
         Expect.pass
 
 
+leftClickEvent : ( String, Json.Encode.Value )
+leftClickEvent =
+    Event.custom "click" <|
+        Json.Encode.object
+            [ ( "ctrlKey", Json.Encode.bool False )
+            , ( "altKey", Json.Encode.bool False )
+            , ( "metaKey", Json.Encode.bool False )
+            , ( "shiftKey", Json.Encode.bool False )
+            , ( "button", Json.Encode.int 0 )
+            ]
+
+
```

Comment:
a bit of a nit, but can this go in the `Data` module? Intuitively it feels like test data more than a helper function. By that argument I guess `pipelineRunningKeyframes` should go in there too, but no pressure.